.gitignore

```diff
@@ -85,7 +85,7 @@ docs/*/pdfreactor-fonts.html
 #################
 *.egg-info
 .installed.cfg
-static/
+/static/
```

Comment:
why is static in ignore? I don't understand the specifics of how cfgov's static assets are setup but why would we want to not checkin the static dir? Is this the collect static dir?